cfgov/preprocessed/js/modules/util/breakpoint-state.js

```diff
@@ -17,7 +17,7 @@ function _inBreakpointRange( breakpointRange, width ) {
 
 /**
  * @returns {object} An object literal with Boolean
- * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ * isBpXs, isBpSm, isBpMed, isBpLg, isBpXl properties.
```

Comment:
I went back and forth, but I honestly don't care either way so I'll update to all cap size.